aten/src/ATen/test/basic.cpp

```diff
@@ -345,3 +345,58 @@ TEST(BasicTest, BasicTestCUDA) {
     test(CUDA(kFloat));
   }
 }
+
+TEST(BasicTest, AAAA) {
```

Comment:
This shows that C++ stays the same

Reply:
I think we should move this test to `test/cpp/api/tensor.cpp`, and use the `bartlett_window` function from `torch::` namespace (with assertion that the output's `.requires_grad()` is false). This would ensure that the C++ API behavior stays the same, as people are using it from the `torch::` namespace.